workshops/rock_paper_scissors/README.md

```diff
@@ -0,0 +1,294 @@
+Name: 'Rock Paper Scissors Game'
+Description: 'Simple rock paper scissors game using Python'
+Author: '@JackTDC'
+---
+
+# Rock Paper Scissors Game
+Have you ever wondered how to build a game with Python? Have you ever wanted to build your own game but didn't because it was hard? Well, today I'm going to show you how to build one and show you just how simple it can be!
+
+![homepage](https://cloud-8p13u30yt.vercel.app/rps.png)
+
+Here's the [live demo][final_live_demo] and [final code][final_code].
+
+[final_live_demo]:https://repl.it/@JackTDC/Rock-paper-Scissors#main.py
+[final_code]: https://repl.it/@JackTDC/Rock-paper-Scissors#main.py
+[repl_it]: https://repl.it
+
+## Part 1: Prerequisites
+
+You should have a beginner understanding of: 
+- Python
+
+## Part 2: Setup
+
+### Setting up your code environment on Repl.it
+
+[Repl.it](https://repl.it) is an online code editor where you can build your game. You don't have to use Repl.it but I suggest you do as it sets everything up for you and you don't require any installations.
+
+To get started, go to [repl.it/languages/python](https://repl.it/languages/python). Your coding environment will spin up in just a few seconds!
+
+You should see something like the following:
+
+![main.py file in Repl.it](https://cloud-lukrtynqs.vercel.app/repl.png)
+
+## Part 3: Making The Game
+![GIF](https://cloud-931ehqeec.vercel.app/coding.gif)
+
+### 1) Importing Modules
+First, we'll need to import the `randint` module. To learn more about the randint package click [here][randint].
+
+[randint]: https://www.journaldev.com/36085/randint-method-in-python/
+
+On the first line of the `main.py` file, type `from random import randint`. This will import the randint module.
+
+### 2) Giving values to the variables
+Let's make a list for all the entries a player could enter:
+```py
+t = ["r","p","s"]
+```
+'r' stands for `Rock`,'p' stands for `Paper` and 's' stands for `Scissors`.
+
+Now, let's write some code that chooses rock, paper or scissors randomly.
+```py
+computer = t[randint(0,2)]
+```
+Now we will set a variable to false and by using it in a `while` loop, we can keep the game playing infinitely! To do that, just type 
+```py
+player = False
+```
+
+Now, let's write some code so the player can enter their name.
+```py
+name = input("Enter your name:")
+```
+
+Also to make a scoring system, we'll write the following code:
+```py
+You = 0
+PC = 0 
+```
+
+Next, we want to give an option to the player to reset the game.
+```py
+print("Type 'reset' to reset score")
+```
+
+Here's the code so far:
+```py
+from random import randint
+
+t = ["r", "p", "s"]
+
+computer = t[randint(0,2)]
+
+player = False
+
+name = input("Enter your name:")
+print("Type 'reset' to reset score")
+
+You = 0
+PC = 0
+```
+
+### 3)Keeping the game in a loop
+![loop](https://cloud-nmra250be.vercel.app/loop.gif)
+
+For this, I'll use a `while` loop but you can also use `for` loops!
+
+So we'll just type:
+```py
+while Player == False: 
+```
```

Comment:
Throughout here, make sure you’re continuously giving context as to where each bit of code is going. I assume this code is going at the end of the file, but it’s easy to get lost if the only instructions are “next, add this”. It’s probably overkill to specify that for every single step, but maybe add that every once in a while.